doc/classes/ProjectSettings.xml

```diff
@@ -1523,6 +1523,11 @@
 			Depending on the specific devices you are targeting, you may want to raise it.
 			[b]Note:[/b] This setting is only meaningful if [code]rendering/gles3/shaders/shader_compilation_mode[/code] is [b]not[/b] [code]Synchronous[/code].
 		</member>
+		<member name="rendering/gles3/shaders/max_simultaneous_compiles.web" type="int" setter="" getter="" default="1">
+			The default is a very conservative override for [code]rendering/gles3/shaders/max_concurrent_compiles[/code].
```

Comment:
The `[code]` actually refers to a wrong name (also in the original `.mobile` description).

Would be better to change all those to use `[member rendering/gles3/shaders/max_simultaneous_compiles]` syntax so it creates a reference and it's validated by CI.

Reply:
Good catches! This is the kind of changes where an additional tetrad of eyes come handy. 😀